src/pages/AddJob.jsx

```diff
@@ -28,7 +28,7 @@ export default function AddJob() {
         ...data,
         company: auth.user.company,
         owner: auth.userRef,
-        status: "open",
+        status: "pending",
         postDate: new Date(),
         // fixme: temp solution: we need an admin to approve the job
         admin: doc(database, "Users", "iKGlSJEUkWQjCHZMQhgrVixVAt42"),
```

Comment:
Can you please remove the admin piece as well from AddJob. As discussed yesterday we don't need jobs to be tied up to a particular person.

Reply:
@nehakumari2307 done!